datasets/fi/parliament/fi_parliament.yml

```diff
@@ -0,0 +1,151 @@
+title: Finland Members of Parliament
+prefix: fi-parliament
```

Comment:
(nit) I think often we use the native term as the prefix (se-riksdag, de-bundestag). "eduskunta" is a bit weird to us English speakers. I think this is fine, so feel free to just resolve. I just noticed and think it might be good to think about for two minutes and be consistent in the future.